pkg/bench/write_client.go

```diff
@@ -0,0 +1,117 @@
+package bench
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/util/spanlogger"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/opentracing-contrib/go-stdlib/nethttp"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	config_util "github.com/prometheus/common/config"
+	"github.com/prometheus/common/version"
+	"github.com/prometheus/prometheus/storage/remote"
+)
+
+const maxErrMsgLen = 512
+
+var UserAgent = fmt.Sprintf("Prometheus/%s", version.Version)
```

Comment:
Shouldn't we use our own user agent here?

Reply:
We can, I'll update this to use benchtool